task/sql-tasks.js

```diff
@@ -178,7 +281,13 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+            COUNT(*) AS 'TotalOfCurrentProducts',
+            SUM(Discontinued) AS 'TotalOfDiscontinuedProducts'

```

Comment:
that works, but it's better to use count explicitly if you need to count records.

Reply:
but in this case it's better to just use 3 sql statements, on without a from-clause
e.g.
```
select 
   (select count(*) from ....) as 'T'
   (select count(*) from ....) as 'TT'
```